lib/rucio/web/rest/flaskapi/v1/accounts.py

```diff
@@ -682,16 +682,23 @@ def post(self, account: str) -> 'ResponseReturnValue':
         identity = param_get(parameters, 'identity')
         authtype = param_get(parameters, 'authtype')
         email = param_get(parameters, 'email')
+
+        issuer = request.environ.get('issuer')
+        vo = request.environ.get('vo')
+
+        if not issuer or not vo:
+            return 'Issuer and VO must be set.', 400
```

Comment:
shouldn't this be `return generate_http_error_flask(400, 'Issuer and VO must be set.')` ?

Reply:
Addressed in latest revision